core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -347,6 +344,7 @@ public boolean isPollingThreadCountOptionVisible() {
         /**
          * Update the {@link ExecutorService} instance.
          */
+        @PostConstruct
```

Comment:
So you have two annotations on the same class. How will the invocation order work in such case? `load()` must be always invoked first

Reply:
not on the same class (this is forbidden by the spec) but on parent and derived class, which is already implemented the same way in spring and stapler by invoking in parent-first order

Reply:
Then it should to the job